docs/Reporting_RabbitMQ_For_Kubernetes_Issues.md

```diff
@@ -0,0 +1,59 @@
+
+## The minimum set of informations to start with
+
+- The type of product been used:
+  - Commercial
+  - Open source  
+- The version of the product been used: 
+  - Commercial: The Tanzu RabbitMQ version used (1.1,1.2,1.3 ecc...)
+  - Open source: The version of the operator and RabbitMQ image used
+- The Kubernetes distribution been used: TKG, GKE, AKS, EKS, Openshift, Anthos ecc...
+- The Kubernetes version been used.
+- The infrastructure where the Kubernetes cluster is hosted If not obvious (as a Iaas on public cloud? on vsphere? Some info about the machines can be  useful as well.
+- If possible all the informations about the Kubernetes cluster deployed: Number of nodes, total cores, memory, Network used Calico, CNI, ecc... Storage classes defined and used ecc..
+- The deployed RabbitMQ cluster informations: How many pods, what Load Balancer, what storage, what plugins are enabled? Best to have the .yaml of RabbitMQ cluster definition file to have all the info easily collected.
+- Detailed description of scenario causing the issue:
+  -  Is an issue happening during installation/upgrade of the product or running a scenario on a correctly deployed RabbitMQ cluster?
+  -  Is the issue happening with a specific scenario: Ex. (standby replication, MQTT, STOMP ecc..)? And is it still reproducible with it? If yes to describe the scenario
+  -  Is the issue impacting a specific Kubernetes operator / commercial functionality? (Cluster operator, Mesagging topology operator, Standby operator) or is about a RabbitMQ core functionality?
+  -  For RabbitMQ core specific scenarios you can look at (https://github.com/rabbitmq/support-tools/blob/master/docs/Reporting_RabbitMQ_Issues.md)
+  -  If still valid for the incident all the logs of the RabbitMQ cluster pods, and operators.
+ 
+## Installation/Upgrade of Tanzu RabbitMQ commercial version specific issues
+
+Tanzu RabbitMQ is installed through the Carvel toolchain.
+As a prerequisites is necessary to deploy on the RabbitMQ cluster the Tanzu Cluster Essential component wich deploy Kapp and secretgen.
+
+### Prerequisites:
+
+- Versions of the tanzu cluster essential been installed or of the kapp and secretgen components. Also cert-manager is necessary so the version of cert-manager used in the Kubernetes cluster.
+- Status, description of the replicaset and pods inside the kapp-controller namespace as well as logs of the pods (kubectl get all, kubectl get, kubectl describe, kubectl logs)
+- Status, description of the replicaset and pods inside the secretgen-controller namespace as well as logs of the pods (kubectl get all, kubectl get, kubectl describe, kubectl logs)
+- Status, description of the PackageRepository and PackageInstall objects. Yaml definition files of the objects can be very useful as well if available (this can be get with kubectl get -o yaml)
```

Comment:
Same here, I would provide ready to pick up commands to gather the information we are requesting. There's a catch with TKG and Kapp controller, I believe it's installed in `tkg-system` or something similar.